app/presenters/summary/sections/passporting_benefit_check_partner.rb

```diff
@@ -14,6 +14,10 @@ def change_path
       def person
         @person ||= crime_application.partner
       end
+
+      def subject_type
```

Comment:
Does this file require the `HasDynamicSubject` concern?

Reply:
It inherits it from `Sections::PassportingBenefitCheck`.